client/src/components/categorical/value/index.js

```diff
@@ -455,6 +455,9 @@ class CategoryValue extends React.Component {
         VALUE_HEIGHT
       ) ?? {};
 
+    if (xScale === undefined || yScale === undefined || bins === undefined)
```

Comment:
Does it have to be `undefined` — why not: `if (!xScale || !yScale || !bins)`?

Reply:
It could, I just chose to be a bit more verbose for clarity.  Especially since the assignment is a little nuanced.:
```js
const { xScale, yScale, bins } =
      this.createHistogramBins(
        metadataField,
        categoryData,
        colorAccessor,
        colorData,
        label,
        CHART_WIDTH,
        VALUE_HEIGHT
      ) ?? {};
```